### PR TITLE
fix https for GATech authenticated cache

### PR DESCRIPTION
**Pull request by zvada**

### CODE REVIEW

**Comment by zvada** [comment]
@DrDaveD, please review when you can.

File: goc/config-osg/etc/cvmfs/config.d/ligo.osgstorage.org.conf [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CVMFS_EXTERNAL_URL="https://xrootd-local.unl.edu:1094//user/ligo/;https://its-condor-xrootd1.syr.edu:8443//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8444//user/ligo/;http://osg-gftp.pace.gatech.edu:8443//user/ligo/"
+CVMFS_EXTERNAL_URL="https://xrootd-local.unl.edu:1094//user/ligo/;https://its-condor-xrootd1.syr.edu:8443//user/ligo/;https://osg-kansas-city-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-chicago-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-new-york-stashcache.nrp.internet2.edu:8444//user/ligo/;https://osg-gftp.pace.gatech.edu:8443//user/ligo/"
```
Comment (DrDaveD) [inline goc/config-osg/etc/cvmfs/config.d/ligo.osgstorage.org.conf:1]: Have you tested the new server URL?  I tried it, and it isn't working for me.   /var/log/messages doesn't give enough detail, but the CVMFS_DEBUGLOG says "http status error code: HTTP/1.1 404 Not found"

Comment (zvada) [reply goc/config-osg/etc/cvmfs/config.d/ligo.osgstorage.org.conf:1]: @efajardo, can you chime in what tests you have done if any?

Comment (efajardo) [reply goc/config-osg/etc/cvmfs/config.d/ligo.osgstorage.org.conf:1]: Hi @DrDaveD 

It was working yesterday so I suspect it is a transitory problem. I am debugging it with the GaTech Admins.

Comment (efajardo) [reply goc/config-osg/etc/cvmfs/config.d/ligo.osgstorage.org.conf:1]: Hi @zvada ,

we had a problem with the x509 proxy on the cache it is workign now.

**Review by DrDaveD — APPROVED** [review]
I confirm that it is working.

**Pull request merged**